packages/edge-login-ui-rn/src/constants/Scenes.ts

```diff
@@ -0,0 +1,21 @@
+export type SceneNames =
+  | 'ChangePasswordScene'
+  | 'ChangePinScene'
+  | 'ChangeRecoveryScene'
+  | 'NewAccountWelcomeScene'
+  | 'NewAccountUsernameScene'
+  | 'NewAccountPasswordScene'
+  | 'NewAccountPinScene'
+  | 'NewAccountTosScene'
+  | 'NewAccountWaitScene'
+  | 'NewAccountReviewScene'
+  | 'LandingScene'
+  | 'LoadingScene'
+  | 'OtpScene'
+  | 'OtpRepairScene'
+  | 'PasswordScene'
+  | 'PinScene'
+  | 'RecoveryLoginScene'
+  | 'ResecurePasswordScene'
+  | 'ResecurePinScene'
+  | 'SecurityAlertScene'
```

Comment:
Since there are no constants in this file, I think it belongs in `types/`, not `constants/`.